packages/functional_widget/README.md

```diff
@@ -217,6 +217,20 @@ There are a few ways to do so:
   Widget example(int foo, String bar) => Container();
   ```
 
+- With parameters on the `@FunctionalWidget` decorator that will export the generated widget for private functions:
+
+  ```dart
+  @FunctionalWidget(public: true)
+  Widget _privateButExportedExample(int foo, String bar) => Container();
+  ```
+
+- With parameters on the `@FunctionalWidget` decorator that will replace the name with a custom name:
+
+  ```dart
+  @FunctionalWidget(name: "CustomName")
```

Comment:
What is the use-case for this?

Reply:
Been requested by an issue in the past. Since I touched the code, I thought to implement it on the go:

https://github.com/rrousselGit/functional_widget/issues/70